tests/ignore.ledger

```diff
@@ -32,3 +34,12 @@ account Assets:Test2
     Assets:Test3                       10.00 EUR
     Equity:Opening-Balance
 
+; L2Bonly begin
+;2014-07-09 event "location" "Paris, France"
+; 2018-09-01 event "location" "Bologna, Italy"
+; L2Bonly end
+
+; 2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
+# 2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
+*2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
```

Comment:
I'm shocked to learn that the last 2 examples work too, but it's Ledger's fault, not yours :-)

Reply:
https://www.ledger-cli.org/3.0/doc/ledger3.html#Commenting-on-your-Journal